internal/server/routes.go

```diff
@@ -429,10 +429,6 @@ func readRequest(c *gin.Context, req interface{}) error {
 	return nil
 }
 
-func init() {
-	gin.DisableBindValidation()
```

Comment:
Our reduced implementation no longer calls validate, so we're able to remove this.